source/turtles/turtle_hero.py

```diff
@@ -0,0 +1,137 @@
+import pygame, random
+
+GREEN = (20, 255, 140)
+GREY = (210, 210, 210)
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+PURPLE = (255, 0, 255)
+BLACK = (0, 0, 0)
+WIDTH = 100
+HEIGHT = 50
+ACC = 600
+TICK = 1/60.0
+SPEED_FAST = 300
+SPPED_SLOW = 150
+
+class Turtle(pygame.sprite.Sprite):
```

Comment:
jakbys bardzo chciala, to możesz spróbować zrobić atrybuty i metody prywatne;
robi się je, dodając dwa podkreślniki po self.
self.dupa -> public
self.__dupa -> private

Reply:
To na końcu, bo nie wiem jeszcze które się przydadzą i czy zrobić gettery i setter i wszystkie na prywatne.